### PR TITLE
Cherry-pick command-line ref improvements

### PR DESCRIPTION
**Pull request by mdlinville**
These commits are needed to be able to update the 1.13.x command-line reference YAML files for the docs.

**- A picture of a cute animal (not mandatory but encouraged)**

baby rhino with mama

### CODE REVIEW

**Comment by mstanleyjones** [comment]
I have no idea about the PowerPC failure...

File: cli/command/node/inspect.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func newInspectCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.format, "format", "f", "", "Format the output using the given Go template")
-	flags.BoolVar(&opts.pretty, "pretty", false, "Print the information in a human friendly format.")
+	flags.BoolVar(&opts.pretty, "pretty", false, "Print the information in a human friendly format")
```
Comment (thaJeztah) [inline cli/command/node/inspect.go]: I think we should revert these here, as they were not in the 1.13.x code base

File: cli/command/service/inspect.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func newInspectCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.format, "format", "f", "", "Format the output using the given Go template")
-	flags.BoolVar(&opts.pretty, "pretty", false, "Print the information in a human friendly format.")
+	flags.BoolVar(&opts.pretty, "pretty", false, "Print the information in a human friendly format")
```
Comment (thaJeztah) [inline cli/command/service/inspect.go]: Same here

File: docs/reference/commandline/container_prune.md [hunk 3]
```diff
@@ -38,7 +44,72 @@ f98f9c2aa1eaf727e4ec9c0283bc7d4aa4762fbdba7f26191f26c97f64090360
 Total reclaimed space: 212 B
 ```
 
-## Related information
+### Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
```
Comment (thaJeztah) [inline docs/reference/commandline/container_prune.md]: Filtering on `docker container prune` was not yet in docker 1.13

File: docs/reference/commandline/container_prune.md [hunk 4]
```diff
@@ -38,7 +44,72 @@ f98f9c2aa1eaf727e4ec9c0283bc7d4aa4762fbdba7f26191f26c97f64090360
...
+`+-00:00` timezone offset at the end of the timestamp.  When providing Unix
+timestamps enter seconds[.nanoseconds], where seconds is the number of seconds
+that have elapsed since January 1, 1970 (midnight UTC/GMT), not counting leap
+seconds (aka Unix epoch or Unix time), and the optional .nanoseconds field is a
+fraction of a second no more than nine digits long.
+
+The following removes containers created more than 5 minutes ago:
+
+```bash
+$ docker ps -a --format 'table {{.ID}}\t{{.Image}}\t{{.Command}}\t{{.CreatedAt}}\t{{.Status}}'
```
Comment (thaJeztah) [inline docs/reference/commandline/container_prune.md]: `--format` was also not in 1.13

File: docs/reference/commandline/cli.md [hunk 5]
```diff
@@ -165,18 +167,25 @@ attach`, `docker exec`, `docker run` or `docker start` command.
...
-    }
-    {% endraw %}
+```json
+{
+  "HttpHeaders": {
+    "MyHeader": "MyValue"
+  },
+  "psFormat": "table {{.ID}}\\t{{.Image}}\\t{{.Command}}\\t{{.Labels}}",
+  "imagesFormat": "table {{.ID}}\\t{{.Repository}}\\t{{.Tag}}\\t{{.CreatedAt}}",
+  "pluginsFormat": "table {{.ID}}\t{{.Name}}\t{{.Enabled}}",
```
Comment (thaJeztah) [inline docs/reference/commandline/cli.md]: this was not in 1.13

File: docs/reference/commandline/cli.md [hunk 6]
```diff
@@ -165,18 +167,25 @@ attach`, `docker exec`, `docker run` or `docker start` command.
...
+  "statsFormat": "table {{.Container}}\t{{.CPUPerc}}\t{{.MemUsage}}",
+  "servicesFormat": "table {{.ID}}\t{{.Name}}\t{{.Mode}}",
```
Comment (thaJeztah) [inline docs/reference/commandline/cli.md]: this was not in 1.13

File: docs/reference/commandline/image_prune.md [hunk 7]
```diff
@@ -62,7 +67,95 @@ deleted: sha256:2c675ee9ed53425e31a13e3390bf3f539bf8637000e4bcfbb85ee03ef4d910a1
 Total reclaimed space: 16.43 MB
 ```
 
-## Related information
+### Filtering
```
Comment (thaJeztah) [inline docs/reference/commandline/image_prune.md]: `docker image prune --filter` was not yet in 1.13

Comment (thaJeztah) [inline docs/reference/commandline/image_prune.md]: So looks like most examples below cannot work :cry:

File: docs/reference/commandline/network_prune.md [hunk 8]
```diff
@@ -16,20 +16,72 @@ Options:
...
+
 WARNING! This will remove all networks not used by at least one container.
 Are you sure you want to continue? [y/N] y
 Deleted Networks:
 n1
 n2
 ```
 
-## Related information
+### Filtering
```
Comment (thaJeztah) [inline docs/reference/commandline/network_prune.md]: `docker network prune --filter` was not yet in 1.13

File: docs/reference/commandline/plugin_ls.md [hunk 9]
```diff
@@ -40,7 +42,67 @@ ID                  NAME                             TAG                 DESCRIP
 69553ca1d123        tiborvass/sample-volume-plugin   latest              A test plugin for Docker   true
 ```
 
-## Related information
+### Filtering
```
Comment (thaJeztah) [inline docs/reference/commandline/plugin_ls.md]: `docker plugin ls --filter` was not yet in 1.13

File: docs/reference/commandline/plugin_ls.md [hunk 10]
```diff
@@ -40,7 +42,67 @@ ID                  NAME                             TAG                 DESCRIP
...
+
+tiborvass/no-remove
+
+$ docker plugin ls --filter enabled=true
+
+NAME                  TAG                 DESCRIPTION                ENABLED
+```
+
+
+### Formatting
```
Comment (thaJeztah) [inline docs/reference/commandline/plugin_ls.md]: `docker plugin ls --format` was not yet in 1.13

File: docs/reference/commandline/ps.md [hunk 11]
```diff
@@ -331,7 +338,45 @@ CONTAINER ID        IMAGE       COMMAND       CREATED             STATUS
 9d4893ed80fe        ubuntu      "top"         10 minutes ago      Up 10 minutes                           test1
 ```
 
-## Formatting
+#### publish and expose
```
Comment (thaJeztah) [inline docs/reference/commandline/ps.md]: the "publish" and "expose" filters were not yet in 1.13

File: docs/reference/commandline/run.md [hunk 12]
```diff
@@ -529,54 +588,15 @@ Use Docker's `--restart` to specify a container's *restart policy*. A restart
 policy controls whether the Docker daemon restarts a container after exit.
 Docker supports the following restart policies:
 
-<table>
```
Comment (thaJeztah) [inline docs/reference/commandline/run.md]: Didn't we have to change this _to_ a HTML table for the YAML to work?

File: docs/reference/commandline/service_ls.md [hunk 13]
```diff
@@ -103,7 +109,36 @@ ID            NAME   MODE        REPLICAS  IMAGE
 0bcjwfh8ychr  redis  replicated  1/1       redis:3.0.6
 ```
 
-## Related information
+### Formatting
```
Comment (thaJeztah) [inline docs/reference/commandline/service_ls.md]: `docker service ls --format` was not yet in 1.13

File: docs/reference/commandline/stack_services.md [hunk 14]
```diff
@@ -62,7 +66,37 @@ The currently supported filters are:
 * name (`--filter name=myapp_web`)
 * label (`--filter label=key=value`)
 
-## Related information
+### Formatting
```
Comment (thaJeztah) [inline docs/reference/commandline/stack_services.md]: `docker stack services --format` was not yet in 1.13

File: docs/reference/commandline/swarm_init.md [hunk 15]
```diff
@@ -133,7 +136,17 @@ Snapshots compact the Raft log and allow for more efficient transfer of the
 state to new managers. However, there is a performance cost to taking snapshots
 frequently.
 
-## Related information
+### `--availability`
```
Comment (thaJeztah) [inline docs/reference/commandline/swarm_init.md]: This option was not yet in 1.13

File: docs/reference/commandline/system_prune.md [hunk 16]
```diff
@@ -64,7 +67,28 @@ deleted: sha256:3a88a5c81eb5c283e72db2dbc6d65cbfd8e80b6c89bb6e714cfaaa0eed99c548
 Total reclaimed space: 13.5 MB
 ```
 
-## Related information
+### Filtering
```
Comment (thaJeztah) [inline docs/reference/commandline/system_prune.md]: `docker system prune --filter` was not yet in 1.13

**Review by thaJeztah — CHANGES_REQUESTED** [review]
found some changes that slipped in from 17.04

**Comment by mstanleyjones** [comment]
Nobody should merge this until it gets +1 from @thaJeztah

File: docs/reference/commandline/run.md (hunk 12 above)
Comment (mstanleyjones) [inline docs/reference/commandline/run.md]: No, that was a different (more complex) table. It was the one that had to do with volume options.

**Comment by mstanleyjones** [comment]
Addressed feedback.

**Comment by thaJeztah** [comment]
This PR needs to be cherry-picked for the YAML generator to work (when using `make`) https://github.com/docker/docker/pull/32215

File: docs/reference/commandline/run.md [hunk 17]
```diff
@@ -144,6 +144,8 @@ Options:
   -w, --workdir string              Working directory inside the container
 ```
 
+## Descriptino
```
Comment (thaJeztah) [inline docs/reference/commandline/run.md]: Oh, there's a PR somewhere that fixes this, but otherwise just add a commit :smile:

**Review by thaJeztah — APPROVED** [review]
LGTM

found one type that you may want to fix, but feel free to merge after that

**Comment by thaJeztah** [comment]
oh, looks like there's a commit missing a sign-off

**Comment by thaJeztah** [comment]
all looks good now 👍

**Pull request merged**